admin/views/test-developer/manage-check-forms.ejs

```diff
@@ -80,14 +83,20 @@
                         <tr>
                             <td>
                                 <div class="multiple-choice">
-                                    <input id="check-form-<%= form._id %>" type="checkbox" name="check-form" value="<%= form._id %>">
-                                    <label for="check-form-<%= form._id %>"><a href="/test-developer/view-form/<%= form._id %>"><%= form.name %> </a></label>
+                                    <input id="check-form-<%= form._id %>"
+                                           type="checkbox"
+                                           name="check-form"
+                                           value="<%= form._id %>"
+                                           tabindex="0"
+                                           aria-label="Tick form <%= form.name %>"
+                                           aria-checked="false">
+                                    <label for="check-form-<%= form._id %>"><a href="/test-developer/view-form/<%= form._id %>" aria-label="Edit form <%= form.name %>"><%= form.name %> </a></label>
                                 </div>
                             </td>
-                            <td>
-                                <%= form.checkWindows.map( cw => { return cw.name;  }).join(', '); %>
+                            <td tabindex="0" aria-label="Assigned to <%= form.checkWindows.map( cw => { return cw.name;  }).join(', ') || 'pending' %>.">
```

Comment:
Can we change this to: form.checkWindows.map( cw => cw.name ).join(', ')
The curly bracers and return should be redundant after the arrow.